test/test_jit.py

```diff
@@ -6295,6 +6295,77 @@ def forward(self, x, hiddens):
         for out, ref in zip(ei_hid, ref_hid):
             torch.testing.assert_allclose(out, ref)
 
+    @unittest.skipIf(TEST_WITH_UBSAN or not torch.fbgemm_is_cpu_supported(),
+                     'Quantized RNN requires FBGEMM. FBGEMM does not play'
+                     ' well with UBSAN at the moment, so we skip the test if'
+                     ' we are in a UBSAN environment.')
+    def test_gru_quantized(self):
```

Comment:
Instead of making an entirely new test, can you just combine the LSTM and GRU tests like what was done for the cells:

https://github.com/pytorch/pytorch/blob/046c4589df15fc4e7b832d594d3d8e3a54cd47c9/test/test_jit.py#L6128